src/cdk/explorer/nodes/getCfnDefinition.ts

```diff
@@ -0,0 +1,61 @@
+import * as fs from 'fs';
+
+/**
+ * @param uniqueIdentifier unique identifier of state machine
+ * @param cdkOutPath cdk.out path
+ * @param stackName name of parent stack 
+ * 
+ * @returns the ASL Json definition string of the state machine construct
+ */
+export function getStateMachineDefinitionFromCfnTemplate(uniqueIdentifier: string, cdkOutPath: string, stackName: string) {
+
+    try {
+        var data = fs.readFileSync(cdkOutPath + `/${stackName}.template.json`, 'utf8');
+        var jsonObj = JSON.parse(data)
+        var resources = jsonObj.Resources
+
+        for (var key of Object.keys(resources)) {
+            if (key === 'CDKMetadata') continue
+
+            var slicedKey = key.slice(0, -8)
+            if (slicedKey === uniqueIdentifier) {
+                jsonObj = jsonObj.Resources[`${key}`].Properties.DefinitionString["Fn::Join"][1]
+                data = JSON.stringify(jsonObj)
+                data = unescape(data)
```

Comment:
Need to decide whether we want to return the data in unescaped format, or provide the unescape function as a separate function that the caller can invoke if they want to.